### PR TITLE
Fix/skeleton skinning

### PR DESCRIPTION
**Pull request by mycheckgithub**

### CODE REVIEW

**Comment by RemiArnaud** [comment]
@mycheckgithub Can you get birdy and Mark to take a close look at this please?
Please provide us with a new export of the same mb Russ reported issues with. 
Thanks!

**Comment by mbarnes-sb** [comment]
It's not clear to me what the search space is or that the `<skeleton>` URI is being used. Please highlight that in the code, thanks.

File: COLLADAMaya/src/COLLADAMayaControllerExporter.cpp [hunk 1]
```diff
@@ -555,15 +583,17 @@ namespace COLLADAMaya
                 SceneElement* foundElement = mDocumentExporter->getSceneGraph()->findElement ( pathes[i] );
 				if (foundElement != NULL && !result)
 				{
-					foundElement->addSkeletonURI(skeletonUri);
-					UppersceneElement = sceneElement;
+					
+					JointRootElement = searchJointRootElement(sceneElement);
+					COLLADASW::URI skeletonRootUri(visualSceneExporter->getSceneElementURI(JointRootElement));
+					foundElement->addSkeletonURI(skeletonRootUri);
```
Comment (mycheckgithub) [inline COLLADAMaya/src/COLLADAMayaControllerExporter.cpp]: From Api Maya we get all influences joint, then i take the local root (from all those joints), then i go upward until it is a joint type (not a node) >> searchJointRootElement function.
So i have the maximum search space, then  i can add all node in joint array (controller) to include all bind pose.

Comment (mbarnes-sb) [inline COLLADAMaya/src/COLLADAMayaControllerExporter.cpp]: Okay then this needs refinement. The skeleton URI can refer to a `<node>` element of either type, not only `JOINT`. It's the root of the search domain, not necessarily an influence itself.

**Pull request merged**